__tests__/app.test.js

```diff
@@ -63,4 +63,39 @@ describe("GET /api", () => {
             expect(endpoints).toEqual(endpointsFile)
         })
     })
+})
+describe("GET /api/articles/:article_id", () => {
+    test("200 - responds with article object with correct properties", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+          const { article } = body
+          expect(article.article_id).toBe(1)
+          expect(article.hasOwnProperty("author")).toBe(true)
+          expect(article.hasOwnProperty("title")).toBe(true)
+          expect(article.hasOwnProperty("article_id")).toBe(true)
+          expect(article.hasOwnProperty("body")).toBe(true)
+          expect(article.hasOwnProperty("topic")).toBe(true)
+          expect(article.hasOwnProperty("created_at")).toBe(true)
+          expect(article.hasOwnProperty("votes")).toBe(true)
+          expect(article.hasOwnProperty("article_img_url")).toBe(true)
```

Comment:
This is ok i guess but have a look at a different matcher like .toMatchObject to make it look better